imap_processing/idex/l1/idex_l1.py

```diff
@@ -1,24 +1,16 @@
-"""
-Decommutate IDEX CCSDS packets.
-
-This module contains code to decommutate IDEX packets and creates xarrays to
-support creation of L1 data products.
-"""
+"""Decommutate IDEX CCSDS packets."""
 
-import dataclasses
 import logging
 from collections import namedtuple
 from enum import IntEnum
 
 import numpy as np
 import space_packet_parser
 import xarray as xr
-from space_packet_parser import parser, xtcedef
 
-from imap_processing import imap_module_directory
-from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
 from imap_processing.cdf.utils import met_to_j2000ns
-from imap_processing.idex import idex_cdf_attrs
+from imap_processing.idex.l0.idex_l0 import decom_packets
```

Comment:
There is a warm fuzzy feeling that I get when I see the removal/simplification of dependencies :joy: